stellargraph/connector/neo4j/graph.py

```diff
@@ -0,0 +1,113 @@
+__all__ = ["Neo4jStellarGraph"]
+
+import numpy as np
+import scipy.sparse as sps
+from ...core.experimental import experimental
+
+
+@experimental(reason="the class is not fully tested and lacks documentation")
+class Neo4jStellarGraph:
+    def __init__(self, graph_db, is_directed=False):
+
+        self.graph_db = graph_db
+        self._is_directed = is_directed
+
+    def nodes(self):
+        # FIXME: don't assume stellargraphs "ID"s are in the Neo4j graph
+        # use id() instead
+        node_ids_query = f"""    
+            CALL apoc.cypher.run(
+                'MATCH (n)
+                RETURN n.ID as node_ids',
+                {{}}
+            ) YIELD value
+
+            RETURN collect(value.node_ids) as node_ids
```

Comment:
I think we can just return the results directly instead of a list of results?

```suggestion
            MATCH (n)
            RETURN n.ID as node_ids
```

Reply:
this just returns one of the node_ids (first? last?), I still do the collection but the apoc call was definitelt unecessary :+1:

Reply:
ah wait sorry i was just doing this wrong!